requirements.txt

```diff
@@ -51,10 +51,14 @@ scipy==1.6.1
 scispacy==0.4.0
 sentence-transformers==0.4.1.2
 spacy==3.0.5
+spacy-transformers==1.0.1
+# spaCy 3.0.5 requires cupy-cuda102 < 9.0.0.
+cupy-cuda102==8.6.0
```

Comment:
I tried installing this package on my Mac and I'm getting this:
```text
$ pip install cupy-cuda102==8.6.0
ERROR: Could not find a version that satisfies the requirement cupy-cuda102==8.6.0
ERROR: No matching distribution found for cupy-cuda102==8.6.0
```

Reply:
Have you a CUDA-enabled GPU on your Mac?

Reply:
As far as I know, there's no CUDA on my Mac.

If this is only a patch for this concrete experiment, then I think it's fine if it only works on systems with a CUDA GPU. Just wanted to point out that if this change makes it into master, then the installation would fail on Macs.

Reply:
If there is no CUDA, then that's why it fails. It needs CUDA.

> it only works on systems with a CUDA GPU

`cupy-cuda102` is needed to train the pipeline. Would you have a suggestion on handling this?

Reply:
That makes me think, maybe we should create 2 `requirements.txt` files (one for `install_requires` and one for `extra_requires`) to avoid anyone having this kind of problem doing `pip install -r requirements.txt`. What do you think ?

Reply:
> maybe we should create 2 `requirements.txt` files (one for `install_requires` and one for `extra_requires`)

I would support this.

This issue is a part of the bigger discussion reported in #280.